src/Vts.Scripting/ScriptHelper.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Plotly.NET.LayoutObjects;
+
+namespace Vts.Scripting;
+
+public static class ScriptHelper
+{
+    /// <summary>
+    /// Helper extension method that returns an array of midpoints, located halfway between the endpoints of the specified range
+    /// </summary>
+    /// <param name="endpointRange">The range of endpoints</param>
+    /// <returns>The corresponding midpoint outputs</returns>
+    public static double[] GetMidpoints(this DoubleRange endpointRange)
+    {
+        var endpoints = endpointRange.AsEnumerable().ToArray();
+        if (endpoints.Length < 2)
+        {
+            return Array.Empty<double>();
+        }
+
+        var midpoints = new double[endpoints.Length - 1];
+        for (int i = 0; i < midpoints.Length; i++)
+        {
+            midpoints[i] = (endpoints[i + 1] + endpoints[i]) / 2;
+        }
+        return midpoints;
+    }
+
+    /// <summary>
+    /// Helper extension method that returns every nth element of the enumerable, starting at the specified skip index
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    /// <param name="values">the values being filtered</param>
+    /// <param name="n">number of values to jump forward at a time</param>
+    /// <param name="skip">number of values to initially skip</param>
+    /// <returns></returns>
+    public static IEnumerable<T> TakeEveryNth<T>(this IEnumerable<T> values, int n, int skip = 0) =>
+            values.Where((_, i) => (i - skip) % n == 0);
+
+    /// <summary>
+    /// Method to create a standard scatter chart from the specified x and y values using Plotly.NET
+    /// </summary>
+    /// <param name="xValues">The values for the x axis.</param>
+    /// <param name="yValues">The values for the y axis.</param>
+    /// <param name="xLabel">The label for the x axis. Optional.</param>
+    /// <param name="yLabel">The label for the y axis. Optional.</param>
+    /// <param name="title">The title of the chart. Optional.</param>
+    /// <returns>A `GenericChart` instance representing the scatter chart.</returns>
+    public static Plotly.NET.GenericChart.GenericChart ScatterChart(double[] xValues, double[] yValues, string xLabel = "", string yLabel = "", string title = "")
```

Comment:
I see that this, LineChart and HeatMap are not being used, will they be used at a later time, can we add unit tests for these later?

Reply:
They're used throughout the demos. I'm not sure how/why we'd unit test them - they're the things we're skipping in our unit tests. They'll just be helper functions at the top of the notebooks to help with plotting.

Reply:
They are greyed out in my solution, possibly because it doesn't build.